README.md

```diff
@@ -103,7 +103,19 @@ $ docker run -d \
 
 * Then start any containers to be proxied as described previously.
 
-Note: If the docker-gen container name is static and you want to explicitly set it, use `-e NGINX_DOCKER_GEN_CONTAINER=nginx-gen`. The same thing is true with the nginx container (`-e NGINX_PROXY_CONTAINER=nginx`).
+Note: 
+If the 3 containers are using using static names, as shown in the examples. The label (`--label com.github.jrcs.letsencrypt_nginx_proxy_companion.docker_gen`) on `nginx` container can be removed and additional evironment variables set on the `letsencrypt-nginx-proxy-companion` container. The variables are `NGINX_DOCKER_GEN_CONTAINER=nginx-gen` and `NGINX_PROXY_CONTAINER=nginx`.
```

Comment:
I removed the -e from the variables.

Reply:
The variables are `NGINX_PROXY_CONTAINER` and `NGINX_DOCKER_GEN_CONTAINER`, `nginx` and `nginx-gen` are their respective value **in the context of this example only**, that should be made clear like in the rest of the doc.

In my opinion the way you wrote it might cause people to believe that the two variables have to be set to `nginx` and `nginx-gen` regardless of the name given to the nginx and docker-gen containers. It should be made clearer that those two values have to be the static names chosen for the nginx and doker-gen containers.